### ISSUE TITLE
Accessibility with screen readers on windows

### ISSUE THREAD

**Issue by Gamulation** [xbmc/xbmc#23105]
Hi. Would it be possible to ad a screen reader integration to Kodi? There is an external addon that already does this, but it's way out of date and does not work on kodi 20, or even 19. A workaround was made for 19, but not 20. It would be a lot easier if the screen reader addon could be implemented into the main Kodi config. This will allow blind and low vision users to use Kodi again without having to worry about the latest version breaking the screen reader.
The screen reader addon that's currently being used by many on Kodi 18 can be found here: https://github.com/ruuk/service.xbmc.tts
I am running windows 10 64-bit and Kodi 18.9
Please look into this, as it would make the program a lot more accessible.

**Comment by fuzzard** [comment]
Please make feature requests on the forums.
